.github/workflows/ci.yaml

```
@@ -34,9 +34,12 @@ jobs:
               scoop bucket add extras
               scoop install fzf ripgrep fd
           - os: ubuntu-latest
+            env:
+              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)